my.css

```diff
@@ -0,0 +1,104 @@
+.header {
+	position: relative;
+	height: 100px;
+	background: -webkit-linear-gradient(180deg, white 63%, black 37%);
+}
+.burgermenu {
+	position: absolute;
+	right: 0;
+	top: 0;
+	width: 70px;
+	height: 70px;
+	background: gray;
+}
+.burger {
+	position: absolute;
+	width: 30px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)